owtf/data/conf/resources.cfg

```diff
@@ -1189,9 +1189,8 @@ SnmpProbeMethods_____SNMP Probing using _____@@@TOOL_METASPLOIT_DIR@@@/msfconsol
 BruteSnmpProbeMethods_____SNMP Probing using _____@@@TOOL_METASPLOIT_DIR@@@/msfconsole -q -x "use auxiliary/scanner/snmp/snmp_login; set RHOSTS @@@host_ip@@@; set RPORT @@@SNMP_PORT_NUMBER@@@; run; exit"
 PassiveOpenS3Buckets_____Grayhat Warfare (Public Buckets)_____https://buckets.grayhatwarfare.com/results/@@@host_name@@@
 SubdomainTakeover_____Subdomain Takeover_____sh @@@FRAMEWORK_DIR@@@/scripts/subdomain_takeover.sh @@@host_name@@@
+IMAP_SMTPinjection_____IMAP_SMTP Injection_____ command-injection-tester --imap --pop3 --smtp @@@host_name@@@
```

Comment:
can you fix the space between `____` and `command-injection-tester`?

Reply:
done